src/app.js

```diff
@@ -74,26 +78,22 @@ app.get('/:game/game-master.html', (req, res, next) => {
 const staticPath = path.resolve(__dirname, '../static/');
 app.use(express.static(staticPath));
 
-app.get('/:game/:sessionId/gm', (req, res) => {
-    res.cookie('sessionId', req.params.sessionId, { httpOnly: true });
-    res.cookie('role', roles.GAME_MASTER, { httpOnly: true });
-    res.cookie('game', req.params.game, { httpOnly: true });
+app.get('/:game/:session/gm', (req, res) => {
+    setQdSpecificCookies(res, req.params.session, roles.GAME_MASTER, req.params.game);
 
     if (req.isAuthenticated()) {
-        return res.redirect(`/${req.params.game}/game-master.html?sessionId=${req.params.sessionId}`);
+        return res.redirect(`/${req.params.game}/game-master.html?session=${req.params.session}`);
     }
 
     res.cookie('returnUrl', req.originalUrl, { httpOnly: true });
     res.redirect(`/${req.params.game}/login.html`);
 });
 
-app.get('/:game/:sessionId', (req, res) => {
-    res.cookie('sessionId', req.params.sessionId, { httpOnly: true });
-    res.cookie('role', roles.PLAYER, { httpOnly: true });
-    res.cookie('game', req.params.game, { httpOnly: true });
+app.get('/:game/:session', (req, res) => {
+    setQdSpecificCookies(res, req.params.session, roles.PLAYER, req.params.game);
 
     if (req.isAuthenticated()) {
-        return res.redirect(`/${req.params.game}/game.html?sessionId=${req.params.sessionId}`);
```

Comment:
bye-bye cached url 😿

Reply:
Say `Hello!` to brand new `/_qd/lvivjs`!